src/web/routes/application/select-address/select-address.js

```diff
@@ -18,6 +19,7 @@ const buildAddressOption = result => ({
 
 const behaviourForGet = () => async (req, res, next) => {
   res.locals.addresses = req.session.postcodeLookupResults.map(buildAddressOption)
+  req.session.nextAllowedStep = '/manual-address'
```

Comment:
Manual address is further in the flow than select-address, therefore this line is needed to prevent the state machine from redirecting the user back to select-address.

Reply:
I don't understand why this is going to the manual address page, shouldn't this skip past it? I guess I don't understand when it should skip the manual page, and when it shouldn't, and where the difference is.

Reply:
It will skip past it normally, this step allows a user to click on the link that says my address is not listed.
The next subtask is to make sure that the user doesn't see manual-address when they've selected an address.